src/options/options.html

```diff
@@ -12,42 +12,42 @@
 
 	<body>
 		<div class="message-container">
-			<div id="messageTips" class="message-box info invisible fade-hide">
+			<div id="messageTips" aria-label="info" class="message-box info invisible fade-hide">
```

Comment:
```suggestion
			<div id="messageTips" aria-label="message tips" class="message-box info invisible fade-hide">
```

Remember the aria label should not specify the style/class, but useful for one who cannot see the visual styles. So I'd suggest `message tips` or `info box: message tips` or something like this.